api/migrations/54-add-discord-integration-enabled.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE ctfnote.settings
+  ADD COLUMN "discord_integration_enabled" boolean NOT NULL DEFAULT FALSE;
+
+GRANT SELECT ("discord_integration_enabled") ON ctfnote.settings TO user_anonymous;
+REVOKE UPDATE ("discord_integration_enabled") ON ctfnote.settings FROM user_admin;
+GRANT UPDATE ("discord_integration_enabled") ON ctfnote.settings TO user_postgraphile;

```

Comment:
This dit you remove the update rights for `user_admin`? Does this still work with changing the iCal password through the admin settings? Or is this only this single column which can only be changed by the API and not even by the admin?